src/ch02/TrafficLight.enum.ts

```diff
@@ -1,5 +1,42 @@
-export enum TrafficLight {
+export enum RawTrafficLight {
     RED,
     YELLOW,
     GREEN
 }
+class Car {
+    stop(): boolean {
+      return false;
+    }
+  
+    drive(): boolean {
+      return true;
+    }
+  }
+const car = new Car();
+export interface TrafficLight{
+    isRed() : boolean,
+    isYellow() : boolean,
+    isGreen() : boolean,
+    updateCar() : boolean
+}
```

Comment:
```suggestion
export interface TrafficLight {
    updateCar() : boolean
}
```
해당 메서드를 제거하면 어떤 결과가 발생할까요? 이 메서드를 제거가 가능한지 대한 여부를 고민 했으면 좋겠습니다.